packages/@okta/vuepress-site/docs/reference/index.md

```diff
@@ -20,17 +20,17 @@ top_links:
 Details on parameters, requests, and rsponses for Okta's API endpoints.
 
 
-Okta APIs generally fall two categories: signing in your users, and managing
-resources in your kta org.
+Okta APIs generally fall into one of two categories: signing in your users or managing
+resources in your Okta org.
 
-## Sign in Your Users
+## Signing in Your Users
 API endpoints to authenticate your users, challenge for factors, recover passwords, and more.
   - The [Authentication Api](/docs/reference/api/authn) controls user accesss to Okta.
   - The [OpenID Connect & OAuth 2.0 API](/docs/concepts/oauth-openid) controls users access to your applications.
 
 [Learn about which APIs to use.](/docs/concepts/oauth-openid/#authentication-api-vs-oauth-20-vs-openid-connect)
 
-## Manage Okta Objects
+## Managing Resources
```

Comment:
Avoid using gerunds (verbs ending in -ing) in titles and headings. We also use sentence case for all headings. 
```suggestion
## Manage resources
```

Reply:
Changed as requested.